### ISSUE TITLE
Connecting to archive.ubuntu.com

### ISSUE THREAD

**Issue by nlogozzo** [Microsoft/BashOnWindows#2047]
-Windows Version: 10.0.14393 (Anniversary Update)
-Ubuntu Version: 14.04

When i try to run a `sudo apt-get install` or `sudo apt install` or `sudo apt-get update` it is always stuck on:
`0% [Connecting to archive.ubuntu.com (91.189.88.149)]` and then i receive the error:
`Err http://archive.ubuntu.com/ubuntu/ trusty/main librecode0 amd64 3.6-21`
  `Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::21). - connect (113: No route to host) [IP: 2001:67c:1360:8001::21 80]`

**Comment by sundhaug92** [comment]
14393 is an anniversary update, tried updating to creators update?

Does curl/wget work?

If not, do you have a proxy?

**Comment by nlogozzo** [comment]
Curl and wget connect to archive.ubuntu.com with no problem

**Comment by nlogozzo** [comment]
It is only apt and apt-get that have a problem and yes I am in the anniversary update

**Comment by jcdenaes** [comment]
I did the Creators update a few days ago and still have the same issue since Anniversary when trying to upgrade WSL (sudo do-release-upgrade)

I just noticed I have the same URL pattern as nlogozzo.
"Err http://archive.ubuntu.com/ubuntu/ xenial-updates/main libllvm3.8 amd64 1:3.8-2ubuntu4" 

- Is it normal to see a space between "ubuntu/" and "xenial-updates/" (or "ubuntu/ trusty/" in nlogozzo's case)
- Should it rather be "ubuntu/dists/xenial/" instead ?

 /etc/apt/sources.list
deb http://archive.ubuntu.com/ubuntu xenial main restricted universe multiverse
deb http://archive.ubuntu.com/ubuntu xenial-updates main restricted universe multiverse
deb http://archive.ubuntu.com/ubuntu xenial-backports main restricted universe multiverse
deb http://security.ubuntu.com/ubuntu xenial-security main restricted universe multiverse

Thanks,

**Comment by jcdenaes** [comment]
Same here but I stumbled on an awkward solution

I unable to upgrade WSL under Anniversary. Same Err.
I did upgrade to Creators and was still unable to upgrade WSL.

I then did update/upgrade apt-get and the install occured at that moment.

Steps followed

$ sudo do-release-upgrade
...
Err http://archive.ubuntu.com/ubuntu/ xenial-updates/main libllvm3.8 amd64 1:3.8-2ubuntu4
  Connection failed [IP: 91.189.88.162 80]
... 

$ sudo pat-get update
...
 some warnings to be ignored
...

$ sudo apt-get upgrade
...
 upgraded WSL while upgrading apt-get
...

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.2 LTS
Release:        16.04
Codename:       xenial

Hope it helps.
